sonic_installer/main.py

```diff
@@ -449,7 +448,8 @@ def setup_swapmem(self):
         with open(swapfile, 'wb') as fd:
             os.posix_fallocate(fd.fileno(), 0, self.swap_mem_size * SWAPAllocator.MiB_TO_BYTES_FACTOR)
         os.chmod(swapfile, 0o600)
-        run_command(f'mkswap {swapfile}; swapon {swapfile}')
+        run_command(['mkswap', swapfile])
```

Comment:
>run_command [](http://example.com/codeflow?start=8&length=11)

The behavior changes. if first command fails, it will exit.

Reply:
I think second command will run regardless of whether or not the first one succeeds? https://stackoverflow.com/questions/25669540/what-is-the-difference-between-double-ampersand-and-semicolon-in-linux

Reply:
The new implementation will exit if the `mkswap` failed. [https://github.com/sonic-net/sonic-utilities/blob/b81734289b7ca361b365cbf1c845d31b0a81657e/sonic_installer/common.py#L32](https://github.com/sonic-net/sonic-utilities/blob/b81734289b7ca361b365cbf1c845d31b0a81657e/sonic_installer/common.py#L32)